packages/now-next/src/index.ts

```diff
@@ -718,6 +718,9 @@ export async function build({
         // with that routing section
         ...rewrites,
 
+        { handle: 'resource' },
```

Comment:
Let's add a comment about what this does

Reply:
We might have to update validation to use `handle: resource`, have you verified this works?

Reply:
It's working in the tests as can see in the i18n-support fixture since we had a TODO for the directory listing which was able to be removed now that `handle: 'resource'` has been added.